components/x-gift-article/src/GiftArticle.jsx

```diff
@@ -242,7 +242,6 @@ const withGiftFormActions = withActions(
 			isGiftUrlCreated: false,
 			isGiftUrlShortened: false,
 			isNonGiftUrlShortened: false,
-			isArticleSharingUxUpdates: false,
```

Comment:
[Observation] In the original PR this file has a few more changes - [to do with setting the month and date](https://github.com/Financial-Times/x-dash/pull/579/files#diff-7082a9e3cb317538577308da954ff347e6c58e103516faaa15ec7c405cba1161R15) - if you could please check if they can be safely removed, and remove them if that's the case.